### PR TITLE
Split audio and custom scripts; add USB doc

### PR DESCRIPTION
**Pull request by BenjaminHCCarr**
This splits out audio and custom script initialization.
This also future proofs a bit. Will prevent mucking in `custom_setup.sh` (which we hope people will do); from breaking the known working audio output designated in `audio_setup.sh`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,20 @@ After that, you can simply speak to Picroft as you would to any Mycroft implemen
 
 # Help and more info
 Check out the project wiki [here](https://github.com/MycroftAI/enclosure-picroft/wiki).  
-
 There's also the general [Documentation](https://docs.mycroft.ai/).
 
+# There are two scripts run on startu
```
Comment (Landrash) [inline README.md]: Missing p from startup

File: README.md [hunk 2]
```diff
@@ -34,6 +34,20 @@ After that, you can simply speak to Picroft as you would to any Mycroft implemen
...
+* audio_setup.sh configures your specific audio setup
```
Comment (Landrash) [inline README.md]: Suggest wrapping the filenames like this for readability 
> `audio_setup.sh`

**Comment by BenjaminHCCarr** [comment]
Any other thoughts? this includes @btotharye  PR #17 as the first pull of my PR #18.

**Review by penrods — APPROVED** [review]
LGTM

**Comment by penrods** [comment]
LGTM

**Pull request merged**